src/crawlee/_utils/requests.py

```diff
@@ -83,7 +83,7 @@ def normalize_url(url: str, *, keep_url_fragment: bool = False) -> str:
 def compute_unique_key(
     url: str,
     method: str = 'GET',
-    payload: bytes | None = None,
+    payload: str | None = None,
```

Comment:
The helper is also used in the SDK - you may want to take `str | bytes | None` for better compatibility.